### ISSUE TITLE
Move Zarr to shared GitHub organization

### ISSUE THREAD

**Issue by alimanfoo** [zarr-developers/zarr#190]
I'm about ready to put out a 2.2 release candidate for zarr. I've updated the release notes in the master branch, can be viewed here: http://zarr.readthedocs.io/en/master/release.html

There are a couple more things that would be nice to have in this release (https://github.com/alimanfoo/zarr/milestone/5) and if I get a bit of time this week I might try to work on them, but I think I should draw a line under this and make a release.

Before 2.2 is released, I thought it might be a good moment to consider moving zarr (and numcodecs) to a shared github organization. I'd like to do this because I think it would help to recognise that a number of people have made important contributions, and also might help to encourage new contributions and shared ownership of the code. Bottom line is that my day job is genetics and my time for software will always be limited, so if zarr is going to have any longevity then it will need some community support.

I think the pydata org could be a good fit alongside xarray, another option would be a dedicated zarr org. I'd welcome thoughts.

cc @shoyer, @mrocklin, @FrancescAlted, @jakirkham

**Comment by mrocklin** [comment]
I have no strong thoughts here.  Any of `alimanfoo`, `pydata`, or `zarr` are fine with me.  I do think it would be valuable to expand the list of committers, but that can be done orthogonally from the organization name decision.

**Comment by jakirkham** [comment]
Thanks for pushing on getting 2.2 out there. This looks like a really good RC.

Agree with everything Matthew already said. Would add that if you think it is time to move this to an org, I'm supportive. Not sure what is involved in a move to `pydata`, but it would be good to determine what those steps would look like to help evaluate the options. Also this would be a good time to iron out what the best practices are for contributing and document them.

**Comment by jakirkham** [comment]
Not to overstep, but I suspect @rabernat would be interested in this.

**Comment by rabernat** [comment]
Our preliminary work suggests that zarr is a viable format for cloud-based storage of big climate datasets. Although it's very early, I can imagine making a push for much wider adoption. However, I do sometimes worry about the longevity and community around zarr. Moving to a shared github org would not in itself solve these issues, but it has symbolic meaning; zarr would not longer be @alimanfoo's personal project. The next, harder step would be to grow the list of active zarr devs. My vote would probably be for a "zarr" organization, but I don't feel strongly about this.

Is there a list somewhere of projects or packages which use zarr? Is it in use in industry at all?

**Comment by alimanfoo** [comment]
Thanks @rabernat. I'd be happy with a "zarr" organization.

There's one more piece of work I'd like to clear (#217) before making a release candidate. I propose that once #217 is merged I create a "zarr" organization and migrate both zarr and numcodecs there. Numcodecs is a standalone package but is designed primarily for use with zarr so I think it's fine if they both live under a "zarr" org. I'll then make PRs with some documentation on how to contribute to those projects. I'll also make a place in the docs where we can collect a list of projects using zarr (we don't have that at the moment, and I'm not aware of any usage outside research). Once the org migration is done, I'll put a 2.2 release candidate on PyPI. How does that sound?
